<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestSQLite</class>
 <widget class="QWidget" name="TestSQLite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestSQLite</string>
  </property>
  <widget class="QTableView" name="id_tab_list">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>471</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_commit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>60</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>提交修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_cancel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>撤销修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_add">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>140</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>添加记录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_delete">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>180</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>删除选中行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_ascend">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>310</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>按id升序排列</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_descend">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>350</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>按id降序排列</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_query">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>400</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="id_btn_return_table">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>440</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>返回全表</string>
   </property>
  </widget>
  <widget class="QLabel" name="id_lab_name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_edit_name">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>410</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
